rotkehlchen/chain/arbitrum_one/modules/umami/balances.py

```diff
@@ -92,7 +93,7 @@ def query_balances(self) -> 'BalancesSheetType':
                         ) for idx in range(4)
                     ],
                     call_order=call_order,
-                    calls_chunk_size=3,  # etherscan throws an error for >3 chunk size.
+                    calls_chunk_size=ARBITRUM_ETHERSCAN_MULTICALL_CHUNK_SIZE,
```

Comment:
somehow 3 looks really low. Can you check if it is actually correct?

Reply:
yeah, tested again and indeed the comment is right.